src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+ abstract class Machine {
```

Comment:
According to the encapsulation principle and common Java conventions, it's a good practice to specify the access modifier for classes. In this case, you probably want to make the `Machine` class public so it can be used in other packages as well.